packages/webview_flutter/README.md

```diff
@@ -31,30 +27,25 @@ dependencies:
 ## Example
 
 ```dart
-import 'dart:io';
 import 'package:webview_flutter/webview_flutter.dart';
 
 class WebViewExample extends StatefulWidget {
+  const WebViewExample({Key? key}) : super(key: key);
+
   @override
   WebViewExampleState createState() => WebViewExampleState();
 }
 
 class WebViewExampleState extends State<WebViewExample> {
-  @override
-  void initState() {
-    super.initState();
-  }
-
```

Comment:
If this part is no longer needed, why don'y you remove them from the example too?

Reply:
Done!